csrankings-h.csv

```diff
@@ -62,6 +62,7 @@ Haïdar Safa,American University of Beirut,https://www.aub.edu.lb/fas/cs/people/
 Haifeng Liu,Zhejiang University,http://mypage.zju.edu.cn/hfliu,oW108fUAAAAJ
 Haifeng Xu,University of Virginia,https://engineering.virginia.edu/faculty/haifeng-xu,nLgg388AAAAJ
 Haifeng Yu,National University of Singapore,https://www.comp.nus.edu.sg/~yuhf,IDxePZ0AAAAJ
+Haijun Lei,Shenzhen University,http://csse.szu.edu.cn/cn/people?haijunleiszu,j8d2i0gAAAAJ

```

Comment:
Web page incorrect. Should be http://csse.szu.edu.cn/en/peoplec2b7.html?haijunleiszu. Google Scholar link is wrong.